cmd/storage-rest-client.go

```diff
@@ -548,6 +528,20 @@ func (client *storageRESTClient) ReadVersion(ctx context.Context, volume, path,
 
 // ReadXL - reads all contents of xl.meta of a file.
 func (client *storageRESTClient) ReadXL(ctx context.Context, volume string, path string, readData bool) (rf RawFileInfo, err error) {
+	// Use websocket when not reading data.
+	if !readData {
+		resp, err := storageReadXLHandler.Call(ctx, client.gridConn, grid.NewMSSWith(map[string]string{
+			storageRESTDiskID:   client.diskID,
+			storageRESTVolume:   volume,
+			storageRESTFilePath: path,
+			storageRESTReadData: strconv.FormatBool(readData),
```

Comment:
```suggestion
			storageRESTReadData: "false",
```


Just easier avoids future mistakes, in-case people move the code around.